docs/source/reference/video_classification.rst

```diff
@@ -0,0 +1,223 @@
+
+.. _video_classification:
+
+####################
+Video Classification
+####################
+
+********
+The task
+********
+
+Typically, Video Classification is used to identify video clips containing a single object.
```

Comment:
Video classification usually refers to action classification in the video clip rather than classifying what object is in the video so you might want to rephrase slightly.

Reply:
Done !